README.md

```diff
@@ -309,6 +310,14 @@ Type: `bool` (default: `false`)
 `disabled=true` disables the dropdown completely. This is useful during form submit events.
 `readOnly=true` makes the dropdown read only, which means that the user can still interact with it but cannot change any of its values. This can be useful for display only forms.
 
+### id
+
+Type: `string`
+
+Specific id for container. The container renders with a default id of `rddts-N` where N is count of the current component rendered.
```

Comment:
dropdown is a single word, so maybe this should be `rdts`?

More on the `-N` suffix below :)